modules/standard/IO.chpl

```diff
@@ -633,8 +633,8 @@ enum iostringformat {
             are terminated by the terminator byte. This value is appropriate
             to store in iostyle.str_style.
  */
-deprecated
-"stringStyleTerminated is deprecated due to returning a deprecated type"
+@unstable
+"stringStyleTerminated is unstable due to returning a value for the unstable type 'iostyle'"
```

Comment:
I think you meant "of the unstable type"?

Reply:
Ditto for the other ones

Reply:
Technically these are values to be used for fields of ``iostyle``. Something like "value of the unstable type" to me suggests something with the type ``iostyle``, when in fact they are generally returning some kind of ``integral``.

Reply:
Then maybe `from` is better?

Reply:
Just force-pushed the text change we discussed offline - were there any other wording concerns?

Reply:
Nah, I think this is good to go!